nav2_bt_navigator/src/navigators/navigate_through_poses.cpp

```diff
@@ -75,9 +78,10 @@ NavigateThroughPosesNavigator::goalReceived(ActionT::Goal::ConstSharedPtr goal)
   auto bt_xml_filename = goal->behavior_tree;
 
   if (!bt_action_server_->loadBehaviorTree(bt_xml_filename)) {
-    RCLCPP_ERROR(
-      logger_, "Error loading XML file: %s. Navigation canceled.",
-      bt_xml_filename.c_str());
+    current_error_code_ = ActionT::Result::FAILED_TO_LOAD_BEHAVIOR_TREE;
```

Comment:
I don't think any of this actually gets sent anywhere since this is in the goal acceptance part of the action, not the goal processing. This can be removed?

Reply:
keeping as discussed.